version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.0.0-SNAPSHOT"
```

Comment:
2.0.0 hasn't been released yet I think? (We still need to address https://github.com/mdsol/mauth-jvm-clients/issues/65#issuecomment-566651734)

Reply:
Rollback to 2.0.0-SNAPSHOT.